orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/get_backup_data_sizes.sls

```diff
@@ -0,0 +1,23 @@
+{% set configure_remote_db = salt['pillar.get']('postgres:configure_remote_db', 'None') %}
```

Comment:
Where do you call this new Salt State?
I do not see the relevant code part.

Reply:
That is in #13550. I wanted to split up the work into logical pieces, it's also easier to review this way.
This PR would allow users to call the logic using Salt commands from within the primary gateway node, so valid functionality is still being added even without the additional CB code.